xdocs/usermanual/component_reference.xml

```diff
@@ -3023,6 +3023,11 @@ See the <code>jmeter.properties</code> file for details.
 </description>
 </component>
 
+<component name="Correlation Response Recorder" index="&sect-num;.3.10"  width="741" height="141" screenshot="correlation_response_recorder.png">
```

Comment:
What is the use of this component ? I don't see it mentioned in Auto-Correlation manual.
Am I missing something ?